### ISSUE TITLE
Date and time picture is taken on is not always displayed correctly

### ISSUE THREAD

**Issue by JesperJanssen** [mvysny/photocloud-frame-slideshow#84]
For a while now the date and time the picture is taken is not always displayed correctly.

I have the impressions that the app is showing the date the file was last changed instead of the date the picture was taken. For a lot of pictures this is the same but for a lot of pictures this is not.

I'm using OneDrive as a source for my photo's.

For example the picture below (don't know if it works by pasting the picture in this message but otherwise I can send you a link to the picture in my onedrive via mail). The app shows 29-Dec-2015 but the picture was taken of 27-July-2014

I don't know if you recently changed something in displaying the Exif data but I think there is a mismatch somewhere.

Still loving your app by the way. It's the only photo display app I recommend to everybordy who needs one.

![p1200042](https://user-images.githubusercontent.com/17165733/37511054-b503f974-28fd-11e8-8bee-10452c247bbe.JPG)

**Comment by mvysny** [comment]
Let me check. I've uploaded the picture to my Dropbox and indeed it shows 29-Dec-2015. If I pause the Slideshow and unwrap the EXIF panel, it says that the photo was captured at 2015:12:29 16:06:50.

Now if I run Linux utility called `exif` on the image, I'll get:
```
--------------------+----------------------------------------------------------
Tag                 |Value
--------------------+----------------------------------------------------------
Manufacturer        |Panasonic
Model               |DMC-TZ5
Orientation         |Top-left
X-Resolution        |180
Y-Resolution        |180
Resolution Unit     |Inch
Software            |Ver.1.0  
Date and Time       |2015:12:29 16:06:50
YCbCr Positioning   |Co-sited
PRINT Image Matching|208 bytes undefined data
Compression         |JPEG compression
Orientation         |Top-left
X-Resolution        |180
Y-Resolution        |180
Resolution Unit     |Inch
YCbCr Positioning   |Co-sited
Exposure Time       |1/125 sec.
F-Number            |f/4,9
Exposure Program    |Normal program
ISO Speed Ratings   |125
Exif Version        |Exif Version 2.21
Date and Time (Origi|2014:07:27 15:54:12
Date and Time (Digit|2014:07:27 15:54:12
Components Configura|Y Cb Cr -
Compressed Bits per | 4
Exposure Bias       |0,00 EV
Maximum Aperture Val|3,44 EV (f/3,3)
Metering Mode       |Pattern
Light Source        |Unknown
Flash               |Flash did not fire, compulsory flash mode
Focal Length        |47,0 mm
Maker Note          |756 bytes undefined data
FlashPixVersion     |FlashPix Version 1.0
Color Space         |sRGB
Pixel X Dimension   |3712
Pixel Y Dimension   |2088
Sensing Method      |One-chip color area sensor
File Source         |DSC
Scene Type          |Directly photographed
Custom Rendered     |Normal process
Exposure Mode       |Auto exposure
White Balance       |Auto white balance
Digital Zoom Ratio  |0,0
Focal Length in 35mm|280
Scene Capture Type  |Standard
Gain Control        |Low gain up
Contrast            |Normal
Saturation          |Low saturation
Sharpness           |Soft
Image Unique ID     |1a3e89a8a04ed2d185f6a877ab7951ae
GPS Tag Version     |2.2.0.0
North or South Latit|S
Latitude            |33, 26, 13,66791
East or West Longitu|E
Longitude           |25, 46, 0,41107
Altitude Reference  |Sea level
Interoperability Ind|R98
Interoperability Ver|0100
--------------------+----------------------------------------------------------
EXIF data contains a thumbnail (5360 bytes).
```

Note that `0x0132 Date and Time` says 2015:12:29 16:06:50 but `0x9003 Date and Time (Original)` and `0x9004 Date and Time (Digitized)` says `2014:07:27 15:54:12` which appear to be the correct information.

Indeed, according to this page: http://www.exiv2.org/tags.html the `0x0132` value is the value when the image was last modified (probably by some kind of photo editing software). Therefore, I should first display the `0x9003` and/or `0x9004` if it's there; if not I'll display `0x0132`.

**Comment by mvysny** [comment]
> Still loving your app by the way. It's the only photo display app I recommend to everybordy who needs one.

Happy to hear that :) I hope the app doesn't crash for you since I've been getting a couple of OutOfMemory crash reports lately...

**Comment by mvysny** [comment]
Fixed in 1.11.2

**Comment by JesperJanssen** [comment]
Thanks for the quick fix. I will check the fix after the app has been updated.

I haven't got any error message the past couple of months.
